gramex/apps/capture/chromecapture.js

```diff
@@ -226,6 +238,11 @@ async function render(q) {
     }
   }
   await page.close()
+  if (url.parse(q.url, true).query._test_disconnect == 1){
```

Comment:
I would suggest using just `if (q._test_disconnect) {`

`_test_disconnect` will get passed as part of `q` itself, not the URL encoded. So this code block can just be

```js
if (q._test_disconnect) {
  console.log('Disconnecting Chrome')
  browser.disconnect()
}
```